src/WebJobs.Extensions.DurableTask/DurableOrchestrationStatusContext.cs

```diff
@@ -0,0 +1,19 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Microsoft.Azure.WebJobs.Extensions.DurableTask
+{
+    // TODO add comments
+    public class DurableOrchestrationStatusContext
```

Comment:
Maybe we can rename this to `OrchestrationStatusQueryResult`?

Reply:
Also, does it need to be public? If it is only used internally, we should make it internal. Otherwise it will be harder to change later since customers might take a dependency on this type.

Reply:
I agree to change the name.
I think that it needs to be public, because this class is returned to the client as JSON by `HttpApiHandler`.

Reply:
If it's returned as JSON, then it doesn't need to be public. It should only be public if users will use it directly in their function code.

Reply:
@cgillum 
This class is the return value of `DurableOrchestrationClient.GetStatusAsync()` on L223, I think that it can not be internal class, is there any other good idea?